x-pack/plugins/cloud_security_posture/public/common/types.ts

```diff
@@ -36,3 +36,39 @@ export interface CspFindingsQueryData {
 }
 
 export type Sort<T> = NonNullable<Criteria<T>['sort']>;
+
+export interface RuleCreateProps {
```

Comment:
would be nice if we can remove some of the `any`s here, at least for useful things like the filters for example

Reply:
Thanks for the suggestion, I added some types that we are using, and as I couldn't find much information on the types we are not using I changed them to unknown, I think it is at least more accurate than `any`, and then we can change later if we start using them.